### DISCUSSION TITLE
ComfyUI - ZLUDA - Package import Support

### DISCUSSION THREAD

**Discussion by ThomasBC1970** [LykosAI/StabilityMatrix#1059] [Ideas]
Hi to all, ComfyUI with Zluda is perfect running on AMD enviroment. Anyway, it would be great, if we can use Stable Matrix to manage a ComfyUI with Zluda like SD.Next (Zluda) implemented. A working repository is available, but always when I import it to SM its generate faults ... Greetings :-)

**Comment by mohnjiles** [comment]
hi, this should be available as of v2.13.0, thanks for the suggestion!
